snapshot/backup.go

```diff
@@ -431,6 +438,13 @@ func (snap *Snapshot) Backup(scanDir string, imp importer.Importer, options *Bac
 	}
 
 	for fileiter.Next() {
+		select {
+		case <-snap.AppContext().GetContext().Done():
+			// Context canceled (e.g., due to client disconnect)
+			fmt.Println("Context canceled: client disconnected")
```

Comment:
Can we log this message to os.stderr instead?

Reply:
removed, this was for debugging purpose.